services/fileServices/MdPageServices/ContactUsPageService.js

```diff
@@ -0,0 +1,56 @@
+const {
+  retrieveDataFromMarkdown,
+  convertDataToMarkdown,
+} = require("@utils/markdown-utils")
+
+const CONTACT_US_FILE_NAME = "contact-us.md"
+const CONTACT_US_DIRECTORY_NAME = "pages"
+
+class ContactUsPageService {
+  constructor({ gitHubService, footerYmlService }) {
+    this.gitHubService = gitHubService
+    this.footerYmlService = footerYmlService
+  }
+
+  async read(reqDetails) {
+    const { content: rawContent, sha } = await this.gitHubService.read(
+      reqDetails,
+      {
+        fileName: CONTACT_US_FILE_NAME,
+        directoryName: CONTACT_US_DIRECTORY_NAME,
+      }
+    )
+    const { frontMatter, pageContent } = retrieveDataFromMarkdown(rawContent)
+    const { content: footerContent } = await this.footerYmlService.read(
+      reqDetails
+    )
+    frontMatter.feedback = footerContent.feedback
```

Comment:
could we add a comment on why we set `frontMatter.feedback` to `footerContent.feedback`? it seems abit unclear to me .-.

also, is it guaranteed that the `frontMatter` has no `feedback` property otherwise this is a destructive update

Reply:
Due to template intricacies, the feedback url is used in the footer and the contact-us page, but only stored in the footer - the contact-us link to the feedback page is taken from the feedback url stored in the `footer.yml` file. A feedback parameter in the contact us front matter would not be used anywhere, so it's okay to remove it even if present. I've added a comment as to why we retrieve footer information in https://github.com/isomerpages/isomercms-backend/pull/349/commits/33bfb3dacb3517079edb9acaba5afacda4b2b9c8

Reply:
could we add the above explanation to the comment so that it's more fully fledged?

Reply:
https://github.com/isomerpages/isomercms-backend/pull/349/commits/3b6ca17dd13e63e6a283354987a50c0a4fd60c2d